test/istanbul/pragmaTests.coffee

```diff
@@ -85,7 +85,7 @@ module.exports = (run) ->
             istanbulStyle = """
                 console.log "hello"
                 ### istanbul ignore else ###
-                if x
+                if x is true
```

Comment:
Why this change?

Reply:
It's for test coverage: the comments get assigned to a different token depending on how many elements follow the `if` keyword